tests/testthat/test-read_capacity_factors.R

```diff
@@ -1,15 +1,15 @@
 test_that("without specified arguments, read_capacity_factors throws
           error", {
   testthat::expect_error(
-    read_capacity_factors(),
+    read_capacity_factors_power(),
     "Must provide 'path'"
   )
 })
 
 test_that("with all arguments set, old and new versions return data.frame", {
   test_path_new <- testthat::test_path("test_data", "capacity_factors_new.csv")
```

Comment:
since it is a csv file I am guessing that we do not have to rename it into capacity_factors_power_new.csv, but just wanted to pin this, just in case

Reply:
good observation, an you are right, the input files can have any name :)